src/ios/orb.yml

```diff
@@ -291,10 +292,25 @@ commands:
               DEVICES_KEY=$(xcrun simctl list -j | jq -r --arg DEVICES_KEY "$IOS_VERSION" '[.runtimes[] | select (.version==$DEVICES_KEY)][0] | .identifier')
             fi
 
-            UDID=$(ruby -e 'DEVICE_KEY=ARGV[0];DEVICE_NAME=ARGV[1].gsub("\\", "");require "json";puts JSON.parse(`xcrun simctl list -j`)["devices"][DEVICE_KEY].select { |device| device["name"] == DEVICE_NAME && (device["isAvailable"] == true || device["availability"] == "available") }.first["udid"]' $DEVICES_KEY "<< parameters.device >>")
+            DEVICE_NAME=$(ruby -e 'NAME=ARGV[0].gsub("\\", "");puts NAME' "$DEVICE_NAME")
```

Comment:
I think the removal of `\\` is related to this:
https://github.com/wordpress-mobile/WordPress-iOS/blob/14a2dabb5adae0aefdd5be90f7b8e7c992a900ff/.circleci/config.yml#L261
but I'm not familiar with why the double backslashes are used there.